pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -299,7 +302,11 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	initializeDirs()
 
 	// Context to pass around
-	getconfigCtx := getconfigContext{localServerMap: &localServerMap{}}
+	getconfigCtx := getconfigContext{
+		localServerMap: &localServerMap{},
+		// default value of currentMetricInterval
+		currentMetricInterval: zedagentCtx.globalConfig.GlobalValueInt(types.MetricInterval),
```

Comment:
Did Run() already pick up GlobalConfig before this? I thought that was further down.

Reply:
Yes @eriknordmark, it's initialized just few lines above [here](https://github.com/lf-edge/eve/blob/096662d011042e27a0196b3cc0096a82aaecdbb2/pkg/pillar/cmd/zedagent/zedagent.go#L292)

Reply:
OK